internal/handlers/handlers.go

```diff
@@ -1 +1,74 @@
 package handlers
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/Yandex-Practicum/go1fl-sprint6-final/internal/service"
+)
+
+func IndexHandler(w http.ResponseWriter, r *http.Request) {
+	file, err := os.Open("index.html")
+	if err != nil {
+		http.Error(w, "file index.html not found", http.StatusInternalServerError)
+		return
+	}
+	defer file.Close()
+
+	w.Header().Set("Content-Type", "text/html; charset=utf-8")
+
+	_, err = io.Copy(w, file)
+	if err != nil {
+		http.Error(w, "error reading the file", http.StatusInternalServerError)
+	}
+}
+
+func UploadHandler(w http.ResponseWriter, r *http.Request) {
+	r.ParseMultipartForm(10 << 20)
+
+	file, handler, err := r.FormFile("myFile")
+	if err != nil {
+		http.Error(w, "error receiving the file", http.StatusBadRequest)
+		return
+	}
+	defer file.Close()
+
+	fileBytes, err := io.ReadAll(file)
+	if err != nil {
+		http.Error(w, "error reading file", http.StatusInternalServerError)
```

Comment:
Тоже клиент плохой, что не отдал весь контент
```suggestion
		http.Error(w, "error reading file", http.StatusBadRequest)
```